session/bootstrap.go

```diff
@@ -405,9 +406,15 @@ func bootstrap(s Session) {
 				zap.Duration("take time", time.Since(startTime)))
 			return
 		}
+
 		// To reduce conflict when multiple TiDB-server start at the same time.
 		// Actually only one server need to do the bootstrap. So we chose DDL owner to do this.
 		if dom.DDL().OwnerManager().IsOwner() {
+			err = meta.InitMetaTable(s.GetStore())
```

Comment:
the metatable is inited twice if it is a new cluster

Reply:
Fix